### ISSUE TITLE
Scan Final Cut Pro Fails

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#472]
```
2017-04-17 13:34:32: ********
2017-04-17 13:34:32: 13:34:32 ERROR:   LuaSkin: hs.webview.usercontent callback error: ...t/src/extensions/cp/finalcutpro/main/TimelineToolbar.lua:124: attempt to index a nil value
stack traceback:
	...t/src/extensions/cp/finalcutpro/main/TimelineToolbar.lua:124: in function <...t/src/extensions/cp/finalcutpro/main/TimelineToolbar.lua:123>
	(...tail calls...)
	...ub/CommandPost/src/extensions/cp/finalcutpro/axutils.lua:164: in function 'cp.finalcutpro.axutils.cache'
	(...tail calls...)
	...andPost/src/extensions/cp/finalcutpro/ui/RadioButton.lua:55: in function 'cp.finalcutpro.ui.RadioButton.isChecked'
	(...tail calls...)
	...st/src/extensions/cp/finalcutpro/main/EffectsBrowser.lua:319: in function 'cp.finalcutpro.main.EffectsBrowser.saveLayout'
	...CommandPost/src/plugins/finalcutpro/timeline/effects.lua:415: in field 'updateEffectsList'
	.../src/plugins/finalcutpro/preferences/scanfinalcutpro.lua:70: in field 'scanFinalCutPro'
	...gins/finalcutpro/welcome/panels/scanfinalcutpro/init.lua:41: in field 'callbackFn'
	...ub/CommandPost/src/plugins/core/welcome/manager/init.lua:166: in function <...ub/CommandPost/src/plugins/core/welcome/manager/init.lua:160>
2017-04-17 13:34:32: ********
```

**Comment by randomeizer** [comment]
Is this a 10.3.3 issue?

**Comment by latenitefilms** [comment]
I'm not actually sure to be honest. I assumed it was, but "Scan Final Cut Pro" seems to be working fine now, so it could just be a timing/other bug?

**Comment by latenitefilms** [comment]
Fixed in #474.
